### DISCUSSION TITLE
Changing from one package to another

### DISCUSSION THREAD

**Discussion by Philaupatte** [MustardChef/WSABuilds#337] [Q&A]
Hi,
Hope this is the right place to ask. I am currently using **WSA_XXXX.X0000.X.0_XXXX_Release-Nightly-MindTheGapps-XX.X.7z.**
If I want to use another package such as **WSA_XXXX.X0000.X.0_XXXX_Release-Nightly-with-magisk-XX.X.XXXX.-stable-MindTheGapps-XX.X.7z.** 
Should I perform a full process (uninstall/install) or an upgrade is enough?
Thanks

**Comment by MustardChef** [comment]
An upgrade would work, but it may break the subsystem. I have seen people attempting the same and it seems to have worked without any issues.
